custom/icds_reports/utils/aggregation_helpers/distributed/child_health_monthly.py

```diff
@@ -261,8 +261,8 @@ def _state_aggregation_query(self, state_id):
                 "ELSE 'unmeasured' END".format(height_eligible)),
             ("zscore_grading_hfa", "gm.zscore_grading_hfa"),
             ("zscore_grading_hfa_recorded_in_month",
-                "CASE WHEN (date_trunc('MONTH', gm.zscore_grading_hfa_last_recorded) = %(start_date)s) "
-                "THEN 1 ELSE 0 END"),
+                "CASE WHEN (date_trunc('MONTH', gm.zscore_grading_hfa_last_recorded) = %(start_date)s) AND {} "
```

Comment:
can you also add this constraint to `zscore_grading_hfa` column.

Reply:
Updated